{"uid":"86141ec12cd315c4","name":"test_springboot_is_up","fullName":"tests.integration.springboot.test_charm#test_springboot_is_up","historyId":"05ea498b665c4ac2ffc0693af99a3032","time":{"start":1761299116384,"stop":1761299117069,"duration":685},"description":"\n    arrange: build and deploy the Springboot charm.\n    act: call the endpoint.\n    assert: the charm should respond with 200 OK.\n    ","descriptionHtml":"<pre><code>arrange: build and deploy the Springboot charm.\nact: call the endpoint.\nassert: the charm should respond with 200 OK.\n</code></pre>\n","status":"failed","statusMessage":"AssertionError: assert False\n +  where False = UnitStatus(workload_status=StatusInfo(current='maintenance', message='Preparing service for restart', reason='', since='24 Oct 2025 09:45:16Z', version='', life=''), juju_status=StatusInfo(current='executing', message='running secret-storage-relation-changed hook', reason='', since='24 Oct 2025 09:45:15Z', version='3.6.11', life=''), leader=True, upgrading_from='', machine='', open_ports=[], public_address='', address='10.1.181.145', provider_id='spring-boot-k8s-0', subordinates={}).is_active","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/paas-charm/paas-charm/tests/integration/springboot/test_charm.py\", line 25, in test_springboot_is_up\n    assert status.apps[spring_boot_app.name].units[spring_boot_app.name + \"/0\"].is_active\nAssertionError: assert False\n +  where False = UnitStatus(workload_status=StatusInfo(current='maintenance', message='Preparing service for restart', reason='', since='24 Oct 2025 09:45:16Z', version='', life=''), juju_status=StatusInfo(current='executing', message='running secret-storage-relation-changed hook', reason='', since='24 Oct 2025 09:45:15Z', version='3.6.11', life=''), leader=True, upgrading_from='', machine='', open_ports=[], public_address='', address='10.1.181.145', provider_id='spring-boot-k8s-0', subordinates={}).is_active","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"pytestconfig","time":{"start":1761298110997,"stop":1761298110997,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1761298110997,"stop":1761298110997,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"juju","time":{"start":1761298110997,"stop":1761298110997,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"spring_boot_app_image","time":{"start":1761299083246,"stop":1761299083246,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"spring_boot_app","time":{"start":1761299083246,"stop":1761299116378,"duration":33132},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1761299116382,"stop":1761299116383,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"http","time":{"start":1761299116383,"stop":1761299116383,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"\n    arrange: build and deploy the Springboot charm.\n    act: call the endpoint.\n    assert: the charm should respond with 200 OK.\n    ","status":"failed","statusMessage":"AssertionError: assert False\n +  where False = UnitStatus(workload_status=StatusInfo(current='maintenance', message='Preparing service for restart', reason='', since='24 Oct 2025 09:45:16Z', version='', life=''), juju_status=StatusInfo(current='executing', message='running secret-storage-relation-changed hook', reason='', since='24 Oct 2025 09:45:15Z', version='3.6.11', life=''), leader=True, upgrading_from='', machine='', open_ports=[], public_address='', address='10.1.181.145', provider_id='spring-boot-k8s-0', subordinates={}).is_active","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/paas-charm/paas-charm/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/paas-charm/paas-charm/tests/integration/springboot/test_charm.py\", line 25, in test_springboot_is_up\n    assert status.apps[spring_boot_app.name].units[spring_boot_app.name + \"/0\"].is_active\nAssertionError: assert False\n +  where False = UnitStatus(workload_status=StatusInfo(current='maintenance', message='Preparing service for restart', reason='', since='24 Oct 2025 09:45:16Z', version='', life=''), juju_status=StatusInfo(current='executing', message='running secret-storage-relation-changed hook', reason='', since='24 Oct 2025 09:45:15Z', version='3.6.11', life=''), leader=True, upgrading_from='', machine='', open_ports=[], public_address='', address='10.1.181.145', provider_id='spring-boot-k8s-0', subordinates={}).is_active","steps":[],"attachments":[{"uid":"6c3ec123bd6cc7be","name":"log","source":"6c3ec123bd6cc7be.txt","type":"text/plain","size":26903}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"http::0","time":{"start":1761299117075,"stop":1761299117075,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1761299117076,"stop":1761299117076,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"juju::0","time":{"start":1761299117079,"stop":1761299117865,"duration":786},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.integration.springboot"},{"name":"suite","value":"test_charm"},{"name":"host","value":"runnervmxu1zt"},{"name":"thread","value":"31321-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.springboot.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"e0aad75fe0a7b72b","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[]}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":3},"items":[{"uid":"bd879e716dbb7481","reportUrl":"../1121//#testresult/bd879e716dbb7481","status":"passed","time":{"start":1761048307903,"stop":1761048338742,"duration":30839}},{"uid":"f601a8485dcbc437","reportUrl":"../1120//#testresult/f601a8485dcbc437","status":"passed","time":{"start":1761044252927,"stop":1761044263585,"duration":10658}}]},"tags":[]},"source":"86141ec12cd315c4.json","parameterValues":[]}